File: Server/app/AuctionFlower.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class AuctionFlower extends Model
{
    /**
     * The table associated with the model.
     *
     * @var string
     */
    protected $table = 'auctions_flowers';
    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'amount',
        'flowers_per_element',
        'price_highest',
        'price_lowest',
        'auction_id',
        'flower_id',
    ];
}

File: Unity/Assets/Scripts/Models/AuctionFlower.cs
using System;

namespace PlantReality
{
    [Serializable]
    public class AuctionFlower : Model
    {
        public int id;
        public int amount;
        public int flowersPerElement;
        public int minPrice;
        public int startPrice;
        public int price;
        public int auctionId;
        public int flowerId;
        public string createdAt;
        public string updatedAt;
    }
}